scanomatic/ui_server_data/js/ccc/CanvasState.js

```diff
@@ -117,25 +118,23 @@ CanvasState.prototype.draw = function () {
   }
 };
 
-CanvasState.prototype.getMouse = function (e) {
-  let element = this.canvas;
+CanvasState.prototype.getMouse = (e) => {
```

Comment:
not sure if using arrow functions on the prototype instead of `function` functions is a trivial change (might need to research that)

Reply:
It was not – this is what caused the tests to fail... Suggested action?

Reply:
The only thing necessary was to name the functions, so I did that.